web/src/pages/assets/index.tsx

```diff
@@ -29,18 +29,19 @@ export default function AssetsPage() {
 
         // Dummy data for now
         await new Promise((resolve) => setTimeout(resolve, 500)); // Simulate API call
-        setStudies([
-          {
-            id: "1",
-            title: "COVID-19 Research Study",
-            description: "A comprehensive study on COVID-19 impacts and treatments",
-          },
-          {
-            id: "2",
-            title: "Cancer Treatment Analysis",
-            description: "Analysis of various cancer treatment methodologies",
-          },
-        ]);
+        setStudies([]);
+        // setStudies([
+        //   {
+        //     id: "1",
+        //     title: "COVID-19 Research Study",
+        //     description: "A comprehensive study on COVID-19 impacts and treatments",
+        //   },
+        //   {
+        //     id: "2",
+        //     title: "Cancer Treatment Analysis",
+        //     description: "Analysis of various cancer treatment methodologies",
+        //   },
+        // ]);
```

Comment:
As said above, assets is a bit messy, just until I get this first stage out of the way.